### PR TITLE
Adding line power designations.

### PR DESCRIPTION
**Pull request by samisaab**
TAI_NETWORK_INTERFACE_ATTR_CURRENT_INPUT_POWER
TAI_NETWORK_INTERFACE_ATTR_CURRENT_POST-VOA_TOTAL_POWER
TAI_NETWORK_INTERFACE_ATTR_CURRENT_PROVISIONED_CHANNEL_POWER

### CODE REVIEW

File: inc/tainetworkif.h [hunk 1]
```diff
@@ -196,6 +196,30 @@ typedef enum _tai_network_interface_attr_t
...
+     */
+    TAI_NETWORK_INTERFACE_ATTR_CURRENT_INPUT_POWER,
+
+    /**
+     * @brief The total current post-VOA RX input power in dBm
+     *
+     * @type #tai_float_t
+     * @flags READ_ONLY
+     */
+    TAI_NETWORK_INTERFACE_ATTR_CURRENT_POST-VOA_TOTAL_POWER,
```
Comment (ishidawataru) [inline inc/tainetworkif.h]: @samisaab Please use `_` instead of `-`

**Comment by ishidawataru** [comment]
@samisaab Thanks a lot for your submission.

Could you explain what is the difference between `Current Provisioned Channel Power` and `Current Output Power` ?

**Comment by samisaab** [comment]
`Current Output Power` is the total power measured from a module's transmitter.  `Current Input Power` is the total combined power of all channels currently being transmitted in a fiber that is connected to a module's RX port.  The `Current Provisioned Channel Power` is the power measured only in the channel that a module is currently set to.  Channel power is also referenced as `Signal Power`.

**Comment by ishidawataru** [comment]
@samisaab Thanks! merging

**Pull request merged**